src/renderer/entities/walletConnect/lib/utils.ts

```diff
@@ -2,6 +2,7 @@ import Client from '@walletconnect/sign-client';
 
 import type { Chain, Wallet, Account } from '@shared/core';
 import { walletUtils } from '@entities/wallet';
+import { networkUtils } from '../../network';
```

Comment:
⚠️ cross import ⚠️
not sure what actually we can do here
let's at least place a comment `// TODO: resolve cross import`